channel-manager-api.md

```diff
@@ -367,6 +367,102 @@ State `1` is sent with specified minLos and/or maxLos. If `minLos` and/or `maxLo
     "to": "2019-10-04"
 }
 ```
+
+##### Closed to Stay and Closed to Arrival 
+
+Both states should be applied by a partner.
+
+```javascript
+{
+    "ratePlanCode": "FF",
+    "spaceTypeCode": "DEL",
+    "state": [
+        2,
+        8,
+        6
+    ],
+    "minLos": null,
+    "maxLos": null,
+    "from": "2020-09-30",
+    "to": "2020-10-06"
+}
+```
+
+##### Closed to Stay and Closed to Departure
+
+This is not an industry standard and should be disregarded by a partner.
+
+```javascript
+{
+    "ratePlanCode": "FF",
+    "spaceTypeCode": "DEL",
+    "state": [
+        2,
+        8,
+        7
+    ],
+    "minLos": null,
+    "maxLos": null,
+    "from": "2020-09-30",
+    "to": "2020-10-06"
+}
+```
+
+##### Closed to Stay with MLos and CLosed to Arrival
+
+This should be applied as no arrivals are possible and reservation minLos is 3 nights and maxLos is 7 nights.
+
+```javascript
+{
+    "ratePlanCode": "FF",
+    "spaceTypeCode": "DEL",
+    "state": [
+        2,
+        6
+    ],
+    "minLos": 3,
+    "maxLos": 7,
+    "from": "2020-09-30",
+    "to": "2020-10-06"
+}
+```
+##### Closed to Stay and Closed to Arrival with Mlos
+
+This is not an industry standard and should be disregarded by a partner.
+
+```javascript
+{
+    "ratePlanCode": "FF",
+    "spaceTypeCode": "DEL",
+    "state": [
+        2,
+        8
+    ],
+    "minLos": 3,
+    "maxLos": 7,
+    "from": "2020-09-30",
+    "to": "2020-10-06"
+}
+```
+##### Closed to Departure and Closed to Arrival with Mlos
+
+This is not an industry standard and should be disregarded by a partner.
```

Comment:
I still think we need to discuss this. I don't really agree with how it's presented the 'It's not industry standard part' 

We should explain what it means in Mews, so that they can make a judgement on how to interpret this